mesonbuild/build.py

```diff
@@ -2146,7 +2146,8 @@ def determine_filenames(self):
                 prefix = ''
                 # Import library is called foo.lib
                 self.import_filename = self.vs_import_filename
-                create_debug_file = True
+                # Debug files(.pdb) is only created with debug buildtype
+                create_debug_file = self.environment.coredata.get_option(OptionKey("debug"))
```

Comment:
Shoukd this be set in tge rus case as well?

Reply:
Actually, it lloks like we should juat have debug files as a possibility unless were using a gcc like compiler…